serving/docker/deepspeed.Dockerfile

```diff
@@ -34,12 +34,12 @@ COPY scripts scripts/
 RUN mkdir -p /opt/djl/conf && \
     mkdir -p /opt/djl/deps
 COPY deepspeed.config.properties /opt/djl/conf/config.properties
-### Install DJL Serving and remove general python executor
+### Install DJLServing and update latest python engine
```

Comment:
Please add a todo here. We can remove in the future version for python engine update